tekton/README.md

```diff
@@ -113,23 +92,23 @@ To use [`tkn`](https://github.com/tektoncd/cli) to run the `publish-tekton-pipel
    kubectl apply -f tekton/resources.yaml
 
    # Change the environment variable to the version you would like to use.
-   # Be careful: due to #983 it is possible to overwrite previous releases.
-   export VERSION_TAG=vX.Y.Z
-   export GIT_RESOURCE_NAME=tekton-pipelines-git-vX-Y-Z
+   export TEKTON_VERSION=vX.Y.Z
+   export TEKTON_RELEASE_GIT_RESOURCE=tekton-pipelines-git-vX-Y-Z
+   export TEKTON_BUCKET_RESOURCE=pipeline-tekton-bucket
    export IMAGE_REGISTRY=gcr.io/tekton-releases
 
    # Double-check the git revision that is going to be used for the release:
-   kubectl get pipelineresource/tekton-pipelines-git-vX-Y-Z -o=jsonpath="{'Target Revision: '}{.spec.params[?(@.name == 'revision')].value}{'\n'}"
+   kubectl get pipelineresource/$TEKTON_RELEASE_GIT_RESOURCE -o=jsonpath="{'Target Revision: '}{.spec.params[?(@.name == 'revision')].value}{'\n'}"
 
    # Execute the release pipeline.
    # By default this will tag the release as Pipelines' latest. If you would like to prevent
    # this from happening add --param=releaseAsLatest="false"
```

Comment:
It did come up as a pre-release this time.

<img width="90" alt="Skärmavbild 2020-06-05 kl  22 32 24" src="https://user-images.githubusercontent.com/1351150/83920479-9709b300-a77c-11ea-8c32-5d96eeeb4e2d.png">

Reply:
I think it always comes at pre-release. This param is for being able to release older fix release (like a 0.12.2 while 0.13.0 has been already released)